index.js

```diff
@@ -1183,7 +1191,7 @@ function nested (laterCode, name, key, location, subKey, isArray) {
 
           // see comment on anyOf about derefencing the schema before calling ajv.validate
           code += `
-            ${index === 0 ? 'if' : 'else if'}(ajv.validate(${require('util').inspect(location.schema, { depth: null, maxArrayLength: null })}, obj${accessor}))
+            ${index === 0 ? 'if' : 'else if'}(ajv.validate(${JSON.stringify(location.schema)}, obj${accessor}))
```

Comment:
I had to change this line because it was producing this output that trigger a syntax error:

```
if(ajv.validate({
  oneOf: [ { type: 'number' }, { type: 'boolean' } ],
  [Symbol(fast-json-stringify.cloned)]: true
}, obj["cs"]))
```

I don't know why this sentence was using `inspect`

I tried `showHidden: false` since I was expecting that symbols would be ignored but nothing:

```

const obj = {
  [Symbol('hello')]: 'world'
}

const a = require('util').inspect(obj, {
  showHidden: false,
  depth: null,
  maxArrayLength: null
})
console.log(a) // { [Symbol(hello)]: 'world' }

```

Reply:
It was just a mistake, we should use JSON.stringify().